fluid-project/handbook.floeproject.org
# build: update dependencies for v1.1.0 release

## Description

This PR updates direct and indirect dependencies to address security vulnerabilities. In addition to updating, the `debug` dependency has been removed altogether.

## Steps to test

1. run `npm ci` to install the dependencies as specified by `package-lock.json`
2. run `npm run build`
3. run `npm run lint`
4. run `npm start`

**Expected behavior:** The install, build and dev server launch processes should run without issue. Once launched, the site should function as before.

## Additional information

In my environment, the `commitlint` package seems not to be functioning currently, so my commit messages are not being checked for compliance with Conventional Commits. After some testing, I confirmed that this is the case from not only the `build/update-deps-GH-97` branch, but also `dev` and `main`.

This issue has been resolved in #85, so one of the branches will need to be updated after the other is merged into `dev` (depending on the order).

Unrelated to this issue, two direct dependencies were left alone in order to avoid getting mired in ES Modules and other significant changes just before release. These deps are `move-file` and `husky`. Husky has been updated in #85, and a separate issue will be filed to address `move-file` and ESM conversion at a later date.

## Related issues

Resolves #97
